extensions/peerjs.js

```diff
@@ -0,0 +1,957 @@
+// Name: PeerJS
+// ID: peerjs
+// Description: A port of the PeerJS library for Scratch.
+// By: MikeDEVTheDucklord
+// Original: MikeDEV
+// License: MIT
+
+/*
+    PeerJS Extension for Scratch
+
+    MIT License
+
+    Copyright (C) 2025 Mike Renaker "MikeDEV".
+
+    Permission is hereby granted, free of charge, to any person obtaining a copy
+    of this software and associated documentation files (the "Software"), to deal
+    in the Software without restriction, including without limitation the rights
+    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+    copies of the Software, and to permit persons to whom the Software is
+    furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in all
+    copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+    SOFTWARE.
+*/
+
+/* eslint-disable */
+// prettier-ignore
+(async function (Scratch2) {
```

Comment:
Why are you using `Scratch2`? It should be `Scratch`.

Reply:
Nor should it be async?

Reply:
This was at one point loading the PeerJS beforehand, but seems like I forgot to remove it.